libs/pika/resource_partitioner/tests/unit/cross_pool_injection.cpp

```diff
@@ -67,17 +67,19 @@ int pika_main()
     {
         sched->add_remove_scheduler_mode(
             // add these flags
-            pika::threads::scheduler_mode(pika::threads::enable_stealing |
-                pika::threads::enable_stealing_numa |
-                pika::threads::assign_work_thread_parent |
-                pika::threads::steal_after_local),
+            pika::threads::scheduler_mode(
+                pika::threads::scheduler_mode::enable_stealing |
+                pika::threads::scheduler_mode::enable_stealing_numa |
+                pika::threads::scheduler_mode::assign_work_thread_parent |
+                pika::threads::scheduler_mode::steal_after_local),
```

Comment:
I _think_ you should be able to remove the outer wrapping `pika::threads::scheduler_mode` since the `operator|` already returns a `scheduler_mode`. Same below.

Reply:
In my opinion `scheduler_mode` is mentioned enough times in that expression that it doesn't add anything. It's like writing `int(int(3) + int(17) - int(5))`.

Reply:
Yeah, agreed on that point and to be honest I think `add_remove_scheduler_mode` is a mistake, and we should only have `add_scheduler_mode` and `remove_scheduler_mode`. Then it's obvious what is being added/removed and more importantly the order in which adding/removing happens is clearer. Not for this PR. I opened an issue: https://github.com/pika-org/pika/issues/455.